src/test/ResourceFixture.cpp

```diff
@@ -28,6 +30,19 @@ ResourceFixture::ResourceFixture() {
     parse(desc);
 }
 
+std::string getOSName() {
+    std::string osName = BOOST_PLATFORM;
+    boost::replace_all(osName, " ", "_");
+    boost::replace_all(osName, "/", "_");
```

Comment:
To be check: maybe we can replace by an empty string to make the osname simpler such as `macos`.

Reply:
Done